test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPackCommandTest.cs

```diff
@@ -5822,6 +5821,106 @@ public void PackCommand_PackIcon_SymbolsPackage_MustNotHaveIconInfo_Suceed(Symbo
             }
         }
 
+        [Fact]
+        public void PackCommand_ProjectFile_PackageIconUrl_WithNuspec_Warns_Suceed()
+        {
+            var nuspecBuilder = NuspecBuilder.Create();
+            var testDirBuilder = TestDirectoryBuilder.Create();
+
+            var projectFileContent =
+@"<Project ToolsVersion='4.0' DefaultTargets='Build' xmlns='http://schemas.microsoft.com/developer/msbuild/2003'>
+  <PropertyGroup>
+    <OutputType>library</OutputType>
+    <OutputPath>out</OutputPath>
+    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
+    <PackageIconUrl>https://test/icon.jpg</PackageIconUrl>
```

Comment:
Can you elaborate why purpose this has?

Reply:
I want to make sure that when you're packing a project with a nuspec, ensure that the iconUrl property is detected from the nuspec instead of the project file.

Reply:
I'd say that's irrelevant. 
Icon is not special property, it's the other way around. 

There's nothing that's being read from the csproj that overwrites the nuspec.

Reply:
What about PackCommand_ProjectFile_PackageIconUrl_WithNuspec_WithPackTask_Warns_Succeeds ?

Reply:
I'm not following :D

Reply:
- `PackCommand_ProjectFile_PackageIconUrl_WithNuspec_Warns_Succeeds`: Test the mix of .csproj + .nuspec; expect messages from nuspec
- `PackCommand_ProjectFile_PackageIconUrl_WithNuspec_WithPackTask_Warns_Succeeds`: Test .csproj + .nuspeck + NuGet Pack targets via a package; expect messages from nuspec

Reply:
I can remove PackCommand_ProjectFile_PackageIconUrl_WithNuspec_Warns_Succeeds but, I sugget to have PackCommand_ProjectFile_PackageIconUrl_WithNuspec_WithPackTask_Warns_Succeeds

Reply:
Will remove (2)